gatsby-node.js

```diff
@@ -1,8 +1,29 @@
 const redirects = require('./redirects')
+const getReleases = require('./src/data/releases')
```

Comment:
Is `/data` the right directory to put this file? I think of the `data` directory as containing `yml` and `json` files. `getReleases` seems more like a utility function 🤷‍♂